concepts/05 Widgets/SelectBox/00 Getting Started with SelectBox/25 Group Data.md

```diff
@@ -0,0 +1,202 @@
+The **SelectBox** can display data grouped by category. To make this possible, use the [DataSource]() component. Its API allows you to [sort](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#sort), [filter](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#filter), [select](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#select), and [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) data. At its core, the **DataSource** has a [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) - an object that keeps data and provides an API to access and modify it. To configure the store, use the **DataSource**'s [store]() object. Specify its [type]() as *"array"*, pass the initial data array to the [data]() field, and set the [key] field. 
```

Comment:
Links to sort, filter, select, and group distract from the narrative. We better remove them. At the same time, the other links are missing.